modules/mapper-extras/src/yamlRestTest/resources/rest-api-spec/test/rank_feature/20_null_value.yml

```diff
@@ -0,0 +1,90 @@
+setup:
+  - skip:
+        version: " - 8.8.99"
+        reason: "null_value parameter was added in 8.9.0"
+
+  - do:
+      indices.create:
+        index: test1
+        body:
+          settings:
+            number_of_replicas: 0
+          mappings:
+            properties:
+              pagerank:
+                type: rank_feature
+                null_value: 15
+              url_length:
+                type: rank_feature
+
+  - do:
+      index:
+        index: test1
+        id: "1"
+        body:
+          pagerank: 10
+          url_length: 50
+
+  - do:
+      index:
+        index: test1
+        id: "2"
+        body:
+          pagerank: null
+          url_length: null
+
+  - do:
+      indices.refresh: {}
+
+---
+"Non positive null_vallue":
+
```

Comment:
It looks like `setup` section is not relevant for this test, as we are not doing any request agains `test1` index. Should we get rid of `setup` section all togehter, and instead have two tests:
1. "Non positive null_vallue"
2. "Search rank_feature with and without no null_value", where we create and add docs to `test1` and do 2 searches against it. 
Each of this test will have its own `skip` section.